nemo/core/classes/exportable.py

```diff
@@ -254,6 +253,7 @@ def _verify_runtime(
             import onnxruntime
         except (ImportError, ModuleNotFoundError):
             logging.warning(f"ONNX generated at {output}, not verified - please install onnxruntime.\n")
+            onnx.checker.check_model(onnx_model, full_check=True)
```

Comment:
This will fail if import fails

Reply:
onnxruntime and onyx are different packages